src/layouts/grid/lv_grid.c

```diff
@@ -65,27 +65,27 @@ static inline const lv_coord_t * get_row_dsc(lv_obj_t * obj)
 {
     return lv_obj_get_style_grid_row_dsc_array(obj, 0);
 }
-static inline uint8_t get_col_pos(lv_obj_t * obj)
+static inline lv_coord_t get_col_pos(lv_obj_t * obj)
 {
     return lv_obj_get_style_grid_cell_column_pos(obj, 0);
 }
-static inline uint8_t get_row_pos(lv_obj_t * obj)
+static inline lv_coord_t get_row_pos(lv_obj_t * obj)
 {
     return lv_obj_get_style_grid_cell_row_pos(obj, 0);
 }
-static inline uint8_t get_col_span(lv_obj_t * obj)
+static inline lv_coord_t get_col_span(lv_obj_t * obj)
 {
     return lv_obj_get_style_grid_cell_column_span(obj, 0);
 }
-static inline uint8_t get_row_span(lv_obj_t * obj)
+static inline lv_coord_t get_row_span(lv_obj_t * obj)
 {
     return lv_obj_get_style_grid_cell_row_span(obj, 0);
 }
-static inline uint8_t get_cell_col_align(lv_obj_t * obj)
+static inline lv_coord_t get_cell_col_align(lv_obj_t * obj)
```

Comment:
`..._align`s should return `lv_grid_align_t`.  Once these are updated please update `get_grid_col_align` and `get_grid_row_align` too.